src/test/java/spoon/test/template/TemplateTest.java

```diff
@@ -667,7 +665,12 @@ public void createTypeFromTemplate() throws Exception {
 		assertNull(genClass.getMethod("someMethod"));
 		assertTrue(generatedIfaceMethod!=generatedClassMethod);
 		assertTrue(generatedClassMethod.isOverriding(generatedIfaceMethod));
-		
+		//contract: check replace in literals
```

Comment:
Implicit is worse than explicit. What if we move this code in the test? ie this responsibility to the caller?

I guess that string substitution should work  outside createTypeFromTemplate, consequently, could you add a test with a template dedicated to string substitution?

thanks